pkg/services/authz/rbac/mapper.go

```diff
@@ -63,6 +63,15 @@ func newMapper() mapper {
 		"iam.grafana.app": {
 			"teams": newResourceTranslation("teams", "id", false),
 		},
+		"query.grafana.app": {
+			"query": translation{
+				resource:  "query",
+				attribute: "uid",
+				verbMapping: map[string]string{
+					utils.VerbCreate: "datasources:query",
```

Comment:
Interesting we create a query :+1:

Reply:
yeah... the API i think is based on the `/api/ds/query` api ( https://grafana.com/docs/grafana/latest/developers/http_api/data_source/#query-a-data-source ), so we were probably bound by how that one works.